generate_tmt_variables.py

```diff
@@ -0,0 +1,15 @@
+import json
```

Comment:
I would be happier if the `generate_tmt_secrets.py` and `generate_tmt_variables.py` would be merged into one parametrized script.